### ISSUE TITLE
Move this repo into hq

### ISSUE THREAD

**Issue by ghost** [dwyl/intellectual-property#7]
Should we move this repo into hq?

**Comment by nelsonic** [comment]
We like having a repo explaining our intellectual property. 👍

**Comment by ghost** [comment]
@nelsonic should all the issues about domains and trademarks be moved here?

**Comment by nelsonic** [comment]
Hi @markwilliamfirth apologies for un-assigning myself form this yesterday without any _explanation_. 

I did not feel like the issue gave sufficient _explanation_ for ***why*** it made sense to move the Intellectual Property discussion/content to the https://github.com/dwyl/hq repo instead of having a _logically separate_ repo. 

![image](https://cloud.githubusercontent.com/assets/194400/23395501/dfb6ddba-fd86-11e6-9f0a-07db5967342d.png)

I _really_ don't mind what gets done here.
I didn't think I _needed_ to be ***assigned*** to the issue/task because I don't think it requires my _unique_ abilities/input to resolve it, whereas the coding tasks currently assigned to me can _only_ be performed by me (_because I'm the one who has put in the time to acquire the knowledge/skill to build things for our clients & internal projects..._)

> Apologies for _closing_ the issue _prematurely_, I ***agree*** that IP should _definitely_ be on the "_agenda_", I just don't know how "_high up_" given that we don't have a _product_ to "_protect_".

### The idea/reason for having a _separate_ Intellectual Property repository is so that we can capture all IP-related questions in a single place rather than "polluting" the HQ or project-specific repos ...

I'm happy for someone `else` to make the decision as to where to put the IP-related "FAQ".
The trademarks _can_ be added here as a `trademarks.md` with lots-a-detail as to why we applied for them and how other people can do the same. 
That info/detail can _also_ be added to "HQ" if that is preferred. I _really_ don't mind. 

I need to focus on: 
+ https://github.com/dwyl/aws-ses-lambda/pull/4
+ https://github.com/dwyl/atm/pull/13
+ and _several_ other (_sadly closed-source_) client-project PRs that are "protected" from public view ... 


# tl;dr

> Thanks _again_ for being _proactive_ about sorting out all the "Ops" issues, ur doing good work! ✅ 

As a "_rule-of-thumb_" for what to do when you are "stuck" for task: ["_sharpen the saw_"](https://en.wikipedia.org/wiki/The_7_Habits_of_Highly_Effective_People#Continuous_Improvements) ...
I think _everyone_ in the community/team/company should be _focussed_ on (_spending at least 1h per day_) learning/practicing their "PETE" knowledge/skills see: https://github.com/dwyl/technology-stack#the-pete-stack so that we _all_ know ***how*** to take a feature/idea from low-fi sketch to production in the shortest amount of time possible.

Even if we don't all _write_ the code, we should all know _how_ to. Even people in "Customer Service" should know how to code. Heck, I would suggest the people doing the _cleaning_ should learn how to code so they can build the cleaning robots that will eventually replace them. 

I spend all my "_spare_" moments learning how to write better code and I would _encourage_ everyone `else` to do the same because it _systematically_ adds more value (_i.e. we can do better/valuable work for the clients/company_). This is the highest-value work we can all do. 
Everything `else` is a "_nice-to-have_" at this point (_after taxes/salaries are paid_). 😉 

> Please only assign issues to me when _only_ I can can answer/resolve them. 
(_if in doubt mention but don't assign..._)
And only add a `priority-1` label when something is either causing us _acute pain_ or costing us money/reputation _right now_ or will do in the _immediate_ future. 
See: https://github.com/dwyl/labels#labels
as @iteles wrote: `priority-1`_ means "**drop everything** and work on this (used only when completely necessary)_"

> if _ever_ in doubt what is a `P1` that `should` be _assigned_ to me (_nelson_), ask: 
"_Is this a **Production Bug**_?".

Thanks! Hope you have a great day! ❤️  🌈  ☀️

**Comment by ghost** [comment]
Resolved to move internal ip issues to hq for now to have them all in the same place. Will make decision regarding where FAQ file goes at a later date - moved to priority 3

**Comment by nelsonic** [comment]
Closing as this is nonsense "busywork" that adds no value to anyone.
